content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -293,17 +294,23 @@ Implicit m-n relations:
 
   </Admonition>
 
-#### Conventions for relation tables in implicit m-n-relations
+#### Conventions for relation tables in implicit m-n relations
 
 If you obtain your data model from [introspection](/concepts/components/introspection), you can still use implicit many-to-many relations by following Prisma's [conventions for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations).
 The following example assumes you want to create a relation table to get an implicit many-to-many relation for two models called `Post` and `Category`.
 
 ##### Table name
 
-The name of the relation table must be prefixed with an underscore:
+The name of a relation table for an implicit-many-to-many relation is constructed from the following parts:
 
-- **Valid**: `_CategoryToPost`, `_MyRelation`
-- **Invalid**: `CategoryToPost`, `MyRelation`
+- It must start with an underscore `_`
+- Then follows the alphabetically lower table name (in this case `Category`)
+- After which `To` connects to the rest of the name
+- Which is the alphavetically higher table name  (in this case `Post`)
+
+In the example the correct table name is `_CategoryToPost`.
+
+(When creating an implicit m-n relation via Prisma itself, you can also [configure the name of the relation table to be different](#configuring-the-name-of-the-relation-table-in-implicit-many-to-many-relations) to be for example `_MyRelationTable`)
```

Comment:
I think the description here was actually misleading until now. The intro for the Conventions section makes this about Introspection - and there you start with _no_ schema file and then is can only pick up table names with _exactly_ this name.

The description of the rules is probably terrible, so that should be rewritten in better English @keerlu.

Reply:
Reworded this, again worth a check to see that it's still correct

Reply:
Yes much better. Left comment on detail on the new version.